README.adoc

```diff
@@ -365,7 +365,7 @@ When committing a guide, put it in a commit by itself and use a commit message t
 
 === Filename
 
-To write a blog entry, create a new textile document with the file extension .textile in the guides/ folder. The name of the file will become the last path in the guide's URL and thus should closely match the guide's title. Since it's part of the URL, the filename should not contain spaces.
+To write a blog entry, creates a new textile document with the file extension .textile in the guides/ folder. The name of the file will become the last path in the guide's URL and thus should closely match the guide's title. Since it's part of the URL, the filename should not contain spaces.
```

Comment:
Same as above - I think this should be "create" as it was before. 

```suggestion
To write a blog entry, create a new textile document with the file extension .textile in the guides/ folder. The name of the file will become the last path in the guide's URL and thus should closely match the guide's title. Since it's part of the URL, the filename should not contain spaces.
```